sdk/cmake/msvc.cmake

```diff
@@ -101,7 +114,9 @@ endif()
 
 # Hotpatchable images
 if(ARCH STREQUAL "i386")
-    add_compile_flags("/hotpatch")
+    if(NOT USE_CLANG_CL)
+        add_compile_flags("/hotpatch")
```

Comment:
too bad :(

Reply:
They don't recognize it yet, but when they will, we'll have this again :)